l10n_br_base/models/res_partner.py

```diff
@@ -21,7 +21,7 @@ class Partner(models.Model):
     _name = "res.partner"
     _inherit = [_name, "l10n_br_base.party.mixin"]
 
-    vat = fields.Char(related="cnpj_cpf")
+    vat = fields.Char()
```

Comment:
Hello @JorgeJuarezCasai overall fixing these little glitches here and here for a multi-countries usage was expected despite I took a great care the overall modules design to make it possible.

Your fix looks good overall, though in this case you should simply remove that vat field declaration (line 24) as no vat field override is required anymore. (the default from the base module is enough).